pkg/bosh/converter/docker_image.go

```diff
@@ -13,6 +13,9 @@ type DockerSource struct {
 // GetName returns the name of the docker image
 // More info: https://kubernetes.io/docs/concepts/containers/images
 func (d DockerSource) GetName() string {
+	if d.Organization == "" {
+		return d.Repository + ":" + d.Tag
+	}
```

Comment:
Is this testable ? (I see no unit tests for docker_image.go at all)

Reply:
Sure, feel free to add some. Otherwise I can backfill them tomorrow.